Cargo.toml

```diff
@@ -37,6 +37,8 @@ hyper-proxy = { version = "0.9.1", default-features = false, features = [
 headers = "0.3"
 futures = "0.3.30"
 pbjson-types = "0.6.0"
+josekit = { git = "ssh://git@github.com/famedly/josekit-rs.git", rev = "5941d0f39d034e9c6d96dc47f391926f5e3038fa" }
```

Comment:
I used this lib because vzd-fhir-dir-proxy needs to use it. I'll migrate the rest of the code later to this lib and remove jsonwebtoken

Reply:
Issue created #37